src/main/java/core/basesyntax/model/FruitTransaction.java

```diff
@@ -0,0 +1,66 @@
+package core.basesyntax.model;
+
+public class FruitTransaction {
+    private Operation operation;
+    private String fruit;
+    private int quantity;
+
+    public FruitTransaction(Operation operation, String fruit, int quantity) {
+        this.operation = operation;
+        this.fruit = fruit;
+        this.quantity = quantity;
+    }
+
+    public Operation getOperation() {
+        return operation;
+    }
+
+    public FruitTransaction setOperation(Operation operation) {
+        this.operation = operation;
+        return this;
+    }
+
+    public String getFruit() {
+        return fruit;
+    }
+
+    public FruitTransaction setFruit(String fruit) {
+        this.fruit = fruit;
+        return this;
+    }
+
+    public int getQuantity() {
+        return quantity;
+    }
+
+    public FruitTransaction setQuantity(int quantity) {
+        this.quantity = quantity;
```

Comment:
Consider adding validation in the `setQuantity` method to ensure that the quantity is not set to a negative value. This will help maintain data integrity.